system_launch/CMakeLists.txt

```diff
@@ -9,6 +9,11 @@ if(BUILD_TESTING)
   ament_lint_auto_find_test_dependencies()
 endif()
 
+if(BUILD_TESTING)
+  find_package(ament_lint_auto REQUIRED)
+  ament_lint_auto_find_test_dependencies()
+endif()
```

Comment:
@jilaada Sorry to comment on an old PR. Is this line a duplicate? (L7-L10 and L12-L15)